lib/kdump_utils.pm

```diff
@@ -27,10 +27,18 @@ our @EXPORT = qw(install_kernel_debuginfo prepare_for_kdump
 
 sub install_kernel_debuginfo {
     zypper_call 'ref';
-    # JeOS uses kernel-default-base, except on aarch64 openSUSE
     my $kernel    = script_output('rpm -qf --qf %{name} /boot/initrd-$(uname -r)');
     my $debuginfo = script_output('rpmquery --queryformat="%{NAME}-%{VERSION}-%{RELEASE}\n" ' . $kernel . '| sort --version-sort | tail -n 1');
     $debuginfo =~ s/$kernel/$kernel-debuginfo/g;
+    # Since SLE15-SP2+/Leap 15.2+ (standard and JeOS) there is 'kernel-default-base' but no 'kernel-default-base-debuginfo'
+    # use 'kernel-default-debuginfo' instead.
+    if ((is_sle('>=15-sp2') || is_leap('>=15.2') || is_tumbleweed) && $kernel eq 'kernel-default-base') {
+        $debuginfo =~ s/-base//g;
+        # kernel-default-base and kernel-default packages shared source and binaries, but can be created independently.
+        # They both share kernel-default-debuginfo package. This causes kernel-default-base to have extra numbers added to the release version.
```

Comment:
The "extra numbers" have nothing to do with debuginfo.

Reply:
Yes, wrong wording. Let me try again.